src/event-helper-listen.js

```diff
@@ -24,32 +31,79 @@
  * @param {!EventTarget} element
  * @param {string} eventType
  * @param {function(!Event)} listener
- * @param {boolean=} opt_capture
+ * @param {Object=} opt_evtListenerOpts
```

Comment:
I could not find any callers that use `capture` actually, most use the native `addEventListener` instead (please confirm). So if we're starting this, maybe we should reinforce using options.